### PR TITLE
Facebook

### PR DESCRIPTION
**Pull request by mami198852**

### CODE REVIEW

**Comment by mami198852** [comment]
Instagram/ mami198852

**Comment by garrettw** [comment]
Please don't spam.

File: CodeHub.iOS/Info.plist [hunk 1]
```diff
@@ -91,5 +91,7 @@
 	<string>Photos can be uploaded to comments created within the app.</string>
 	<key>NSCalendarsUsageDescription</key>
 	<string>Calendar is never used within this app.</string>
+	<key>NSPhotoLibraryAddUsageDescription</key>
+	<string>Photos can be saved from readable content within the app.</string>
 </dict>
 </plist>
```
Comment (vanessa-techno) [inline CodeHub.iOS/Info.plist]: Read all

Comment (vanessa-techno) [reply CodeHub.iOS/Info.plist]: []()

Comment (EliteKnight1120) [reply CodeHub.iOS/Info.plist]: https://www.facebook.com/kyle.stege

Comment (EliteKnight1120) [reply CodeHub.iOS/Info.plist]: Md5

Comment (EliteKnight1120) [reply CodeHub.iOS/Info.plist]: Termux

Comment (EliteKnight1120) [reply CodeHub.iOS/Info.plist]: Linux

**Comment by garrettw** [comment]
@thedillonb these were already your commits in the first place. Presumably you had already merged them in at some point. Hopefully merging them now didn't cause any issues.

**Comment by thedillonb** [comment]
@garrettw No harm done. I just noticed I didn't update the `hotfixes` branch with the `master` which had the changes. Instead of accepting the PR I just simply updated the branches and it closed this.

File: CodeHub.iOS/Info.plist (hunk 1 above)
Comment (vanessa-techno) [inline CodeHub.iOS/Info.plist]: Image

File: CodeHub.iOS/Views/SourceTitleView.cs [hunk 2]
```diff
@@ -78,6 +78,7 @@ public override void LayoutSubviews()
             _label.SizeToFit();
             var labelFrame = _label.Frame;
             labelFrame.Height = _label.Frame.Height > halfHeight ? halfHeight : _label.Frame.Height;
+            labelFrame.Width = labelFrame.Width >= Frame.Width ? Frame.Width - 16f : labelFrame.Width;
```
Comment (vanessa-techno) [inline CodeHub.iOS/Views/SourceTitleView.cs]: Image

File: CodeHub.iOS/Views/SourceTitleView.cs [hunk 3]
```diff
@@ -78,6 +78,7 @@ public override void LayoutSubviews()
...
             _label.Frame = labelFrame;
             _label.Center = new CoreGraphics.CGPoint(Frame.Width / 2f, _label.Frame.Height / 2f + (small ? 2f : 3f));
```
Comment (vanessa-techno) [inline CodeHub.iOS/Views/SourceTitleView.cs]: body

File: CodeHub.iOS/Views/SourceTitleView.cs (hunk 2 above)
Comment (vanessa-techno) [reply CodeHub.iOS/Views/SourceTitleView.cs]: Image

File: CodeHub.iOS/Views/SourceTitleView.cs [hunk 4]
```diff
@@ -78,6 +78,7 @@ public override void LayoutSubviews()
             _label.SizeToFit();
             var labelFrame = _label.Frame;
             labelFrame.Height = _label.Frame.Height > halfHeight ? halfHeight : _label.Frame.Height;
+            labelFrame.Width = labelFrame.Width >= Frame.Width ? Frame.Width - 16f : labelFrame.Width;
             _label.Frame = labelFrame;
```
Comment (vanessa-techno) [inline CodeHub.iOS/Views/SourceTitleView.cs]: Image

File: CodeHub.iOS/Views/SourceTitleView.cs (hunk 3 above)
Comment (vanessa-techno) [inline CodeHub.iOS/Views/SourceTitleView.cs]: We are just our new favorite game for sure but

File: CodeHub.iOS/Views/SourceTitleView.cs [hunk 5]
```diff
@@ -78,6 +78,7 @@ public override void LayoutSubviews()
...
             _label.Center = new CoreGraphics.CGPoint(Frame.Width / 2f, _label.Frame.Height / 2f + (small ? 2f : 3f));
```
Comment (vanessa-techno) [inline CodeHub.iOS/Views/SourceTitleView.cs]: “”

**Comment by garrettw** [comment]
@vanessa-techno Stop.

**Pull request merged**